src/drawtypes/label.cpp

```diff
@@ -64,15 +71,57 @@ namespace drawtypes {
    * Create a label by loading values from the configuration
    */
   label_t load_label(const config& conf, string section, string name, bool required, string def) {
+    vector<struct bounds> bound;
+    size_t start, end, pos;
+
     name = string_util::ltrim(string_util::rtrim(name, '>'), '<');
 
     string text;
+    string line{text};
```

Comment:
This would need to be moved down a few line until after `text` has been initialized.